src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -208,8 +211,8 @@ public void canCreateAnInstanceWithoutProvidingID()
     JsonArray identifiers = instanceFromGet.getJsonArray("identifiers");
     assertThat(identifiers.size(), is(1));
     assertThat(identifiers, hasItem(identifierMatches(UUID_ISBN.toString(), "9781473619777")));
-    assertThat(
-      instanceFromGet.getString(STATUS_UPDATED_DATE_PROPERTY), nullValue());
+    assertThat(instanceFromGet.getString(STATUS_UPDATED_DATE_PROPERTY), notNullValue());
+    assertThat(instanceFromGet.getString(STATUS_UPDATED_DATE_PROPERTY), hasIsoFormat());
```

Comment:
Does `hasIsoFormat` allow null as a valid value?

If it does not, could the `notNullValue` above be redundant?

Reply:
I assumed it allows null based on how the existing tests are written-when the original test-writer is testing for a value in that field, they always test for null first and then format-so I do as well.  But I can test and make sure.

Reply:
hasIsoFormat will not accept a null value-and it will tell you if the value was null.  so yes, the null checks are redundant, I will remove.